modules/xmpp/xmpp.js

```diff
@@ -164,6 +165,8 @@ export default class XMPP extends Listenable {
             shard: options.deploymentInfo.shard
         });
 
+        this.connection.moderator = new Moderator(this);
```

Comment:
I'm wondering what is the reason to put the moderator as part of the connection and not directly in the xmpp instance since we are creating it there?

Reply:
connection as XMPP connection? Or JitsiConnection?

Reply:
We need to pass xmpp to Moderator, and in XmppConnection we pass just options no instance of other objects.
We need the xmpp.eventEmitter in Moderator.